modules/cert-manager-acme-dns01-ambient-gcp.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-dns01-ambient-gcp_{context}"]
+= Configuring an ACME issuer to solve DNS-01 challenges by using ambient credentials on GCP
+
+You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using ambient credentials on GCP. This procedure uses _Let's Encrypt_ as the ACME CA server and shows how to solve DNS-01 challenges with Google CloudDNS.
+
+.Prerequisites
+
+* You have access to a role that has permissions to create issuers and certificates in the desired namespace.
+* If you are using GCP with workload identity, you followed the instructions from the _Configuring cloud credentials for the cert-manager Operator for Red Hat OpenShift with GCP Workload Identity_ section.
+* If you are using GCP without workload identity, you followed the instructions from the _Configuring cloud credentials for the cert-manager Operator for Red Hat OpenShift on GCP_ section.
```

Comment:
No links for these references are shown. Same for other places of `you followed the instructions ...`

Reply:
Added additional resources section in the assembly to include section links https://github.com/openshift/openshift-docs/pull/59702/files#diff-1fa7438f3b35bcfae2a4c11d374f7c12ecdf578134a7cf3689d2d5ea6c4f7726R38